librz/core/cmd/cmd_analysis.c

```diff
@@ -7998,3 +7941,35 @@ RZ_IPI RzCmdStatus rz_analyze_value_to_maps_handler(RzCore *core, int argc, cons
 	rz_core_analysis_value_pointers(core, state->mode);
 	return RZ_CMD_STATUS_OK;
 }
+
+RZ_IPI RzCmdStatus rz_analysis_info_show_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
+	cmd_address_info(core, core->offset, state);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_global_imports_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
+	char *imp;
+	RzListIter *iter;
+
+	if (RZ_STR_ISEMPTY(argv[1])) {
+		rz_list_foreach (core->analysis->imports, iter, imp) {
+			switch (state->mode) {
+			case RZ_OUTPUT_MODE_STANDARD:
+				rz_cons_printf("%s\n", imp);
+				break;
+			default:
+				rz_warn_if_reached();
+				break;
+			}
+		}
+	} else {
+		rz_analysis_add_import(core->analysis, argv[1]);
+	}
```

Comment:
how about doing this?
```suggestion
	if (RZ_STR_ISNOTEMPTY(argv[1])) {
		rz_analysis_add_import(core->analysis, argv[1]);
	}
	rz_list_foreach (core->analysis->imports, iter, imp) {
		switch (state->mode) {
		case RZ_OUTPUT_MODE_STANDARD:
			rz_cons_printf("%s\n", imp);
			break;
		default:
			rz_warn_if_reached();
			break;
		}
	}
```

Reply:
Thanks. :-)